src/main/java/org/owasp/wrongsecrets/challenges/ChallengesController.java

```diff
@@ -65,6 +72,20 @@ private boolean checkId(int id) {
         return true;
     }
 
+    @RestController
+    public class secretKey {
+
+        @GetMapping("/challenge/{id}/secret")
```

Comment:
Ehh can you move the restcontroller to a separate class and just not overload all :-)?